connector/src/yang/connector/grpc/telegraf.py

```diff
@@ -0,0 +1,121 @@
+import logging
+import socket
+import configparser
+import subprocess
+import tempfile
+
+from pyats.connections import BaseConnection
+from pyats.easypy import runtime
+from pyats.utils.secret_strings import to_plaintext
+from unicon import Connection
+
+from . import Grpc
+
+# create a logger for this module
+log = logging.getLogger(__name__)
+
+
+class Grpc(Grpc):
+    """
+    Subclass for using Telegraf to collect telemetry
+    """
+    @property
+    def connected(self):
+        """Return True if session is connected."""
+        return True
+
+    def connect(self):
```

Comment:
Please add docstring explaining what the connect method does, including launch of subprocess and CLI connection configuring the device.